roles/config/tasks/patches-manifests/deploy-oci-ccm.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Patch | OCI | CCM | Set namespace oci_ccm_namespace
+  ansible.builtin.set_fact:
+    # default provided by repo is kube-system
+    oci_ccm_namespace: oci-cloud-controller-manager
+  when: oci_ccm_namespace is not defined
+
+- name: Patch | OCI | CCM | Create Namespace
+  ansible.builtin.template:
+    src: patches/oci/oci-ccm-00-namespace.yaml.j2
+    dest: "{{ config_install_dir }}/manifests/oci-cloud-controller-manager-00-namespace.yaml"
+    mode: 0644
+
+- name: Patch | OCI | CCM | Set subnet ID
+  ansible.builtin.set_fact:
+    _lb_subnet1: "{{ sb.state.id }}"
+  loop: "{{ (cluster_state.networks | first).subnets }}"
```

Comment:
This should be improved to query by public/private subnets (use vars to define it)